### DISCUSSION TITLE
如何开启本地搜索功能？

### DISCUSSION THREAD

**Discussion by openjz** [CaiJimmy/hugo-theme-stack#647] [Q&A]
我在本地测试的时候发现搜索功能不返回任何结果，请问怎么开启本地搜索功能？

**Comment by CaiJimmy** [comment]
请上传网站源代码到 GitHub 并提供仓库链接。没有代码我不知道到底哪里出问题 🤷‍♂️

Please upload the website source code to GitHub and provide a link to the repository. Without the code, I won't be able to figure out what's wrong 🤷‍♂️

**Comment by yukexi** [comment]
我的问题是，本地http://localhost:1313/测试搜索功能正常，部署在GitHub page后搜索就成摆设，没用了。不知道问题出在哪？

**Reply by DowneyFlyfan** [reply]
对对对，我也是这个问题
